README.md

```diff
@@ -0,0 +1,24 @@
+# Pokepay Flutter SDK
+## API Document reference
+Please refer to [this document](https://docs.pokepay.jp/guidelines/app-sdk/java.html#_client_api)
+## Install
+1. In the pubspec.yaml of the project, add the following:
+ ```yaml  
+ dependencies:
+    pokepay_flutter_sdk: ^YOUR_VERSION
+ ```
+2. Then run this
+```sh
+flutter pub get
+```
+3. Then you can use Pokepay SDK in your project.
+## Usage
+1. How to use the client

```

Comment:
maybe it would be nice to add the OAuthClient example, how to get the access token. because its the first & required thing to implement.